bin/dist_build.sh

```diff
@@ -121,11 +121,8 @@ FPMCOMMON=(
     --before-install $TEMPLATEDIR/install/before_install.sh
     --after-install $TEMPLATEDIR/install/post_install.sh
     --after-remove $TEMPLATEDIR/install/post_remove.sh
-)
-[ -z $PKGCONFLICTS ] || FPMCOMMON+=( --conflicts $PKGCONFLICTS )
-FPMRPM=(
-    --before-upgrade $TEMPLATEDIR/install/post_remove.sh
```

Comment:
Why this hooks was removed?

Reply:
Since it was never used and `--{before,after}-upgrade` is incompatible with the triggers as we use them. From `fpm -h`:
```
   --after-upgrade FILE          A script to be run after package upgrade. If not specified,
                                  --before-install, --after-install, --before-remove, and 
                                  --after-remove will behave in a backwards-compatible manner
                                  (they will not be upgrade-case aware).
                                  Currently only supports deb, rpm and pacman packages.
    --before-upgrade FILE         A script to be run before package upgrade. If not specified,
                                  --before-install, --after-install, --before-remove, and 
                                  --after-remove will behave in a backwards-compatible manner
                                  (they will not be upgrade-case aware).
                                  Currently only supports deb, rpm and pacman packages.
```